tests/README.md

```diff
@@ -62,3 +62,5 @@ it is both tested as absolute and relative tolerance.
  - test021: hdf5Fast file output and restarting hdf5 simulations
  - test022: Test of specified_psd_c option, LFP homog
  - test023: CIET for LFP
+ - test028: CCCVCPcycling for full cell with LIONSIMBA test file
+ - test029: CCCVCPcycling for half cell with LIONSIMBA test file
```

Comment:
are we adding any test cases for the solid electrolyte? and for the parallel jobs

Reply:
test025 and test026 are for solid electrolyte (at least those are two we added). Let me know if you want me to add that to the tests readme file.